packages/adapter-cloudflare/README.md

```diff
@@ -66,7 +66,8 @@ declare namespace App {
 +		};
 +		context: {
 +			waitUntil(promise: Promise<any>): void;
-+		}
++		};
++		caches: CacheStorage & { default: Cache }
```

Comment:
I wonder if we should show these in alphabetical order?

Reply:
I put `caches` at the bottom because it is the least likely to be used.